src/display/editor/editor.js

```diff
@@ -995,8 +995,8 @@ class AnnotationEditor {
    * new annotation to add to the pdf document.
    *
    * To implement in subclasses.
-   * @param {boolean} isForCopying
-   * @param {Object} [context]
+   * @param {boolean} _isForCopying
+   * @param {Object} [_context]
    */
   serialize(_isForCopying = false, _context = null) {
```

Comment:
This *really* doesn't seem correct, and if anything the parameter-names themselves should be updated instead.
```suggestion
   * @param {boolean} isForCopying
   * @param {Object} [context]
   */
  serialize(isForCopying = false, context = null) {
```